tests/0.8.1/invalid.json

```diff
@@ -0,0 +1,842 @@
+    "assertions": {
+      "header": {
+        "alg": "EdDSA",
+        "typ": "JWT",
+        "ucv": "0.8.1"
+      },
+      "payload": {
+        "iss": "did:key:z6MkgaAaVSzE9XqanrbFD1J9RGTFcWU7N9BPgjSWm17pk1mB",
+        "aud": "did:key:z6MkrjzAyNQV2HmFRiLhNoiTW7RZ76JUTnuHtfsTN1RK6ZPT",
+        "exp": 4803890552,
+        "att": [],
+        "prf": [
+          "eyJhbGciOiJFZERTQSIsInR5cCI6IkpXVCIsInVjdiI6IjAuOC4xIn0.eyJpc3MiOiJkaWQ6a2V5Ono2TWtoNUtnU0hXVnVYSlJMaFFhNjNrSFBSVTVpZllFREc0akw0WjJqcmJlSEhZcCIsImF1ZCI6ImRpZDprZXk6ejZNa21DV2g1aEFZbXM1Zm5VMVNoQkhCTmFVM00xQmVveVlncXJRcGZob255NFBnIiwiZXhwIjo0ODAzODkwNTUyLCJhdHQiOltdLCJwcmYiOltdfQ.dCY2SIBTaXOTKVHAbvQ9Pl1rN3TRHqS9juJXGuXy2uALqRwoyLdcQS1BdUlqEFfLOxBA4aon1pJmw9oTE4HZDQ"
+        ]
+      },
+      "validationErrors": [
+        "prfWitnessNotAligned"
+      ]
+    }
+  },
+  {
+    "comment": "Witness UCAN version does not match delegated UCAN version",
+    "token": "eyJhbGciOiJFZERTQSIsInR5cCI6IkpXVCIsInVjdiI6IjAuOC4xIn0.eyJpc3MiOiJkaWQ6a2V5Ono2TWtnYUFhVlN6RTlYcWFucmJGRDFKOVJHVEZjV1U3TjlCUGdqU1dtMTdwazFtQiIsImF1ZCI6ImRpZDprZXk6ejZNa3JqekF5TlFWMkhtRlJpTGhOb2lUVzdSWjc2SlVUbnVIdGZzVE4xUks2WlBUIiwiZXhwIjo0ODAzODkwNTUyLCJhdHQiOltdLCJwcmYiOlsiZXlKaGJHY2lPaUpGWkVSVFFTSXNJblI1Y0NJNklrcFhWQ0lzSW5WamRpSTZJakF1TnlKOS5leUpwYzNNaU9pSmthV1E2YTJWNU9ubzJUV3R3ZW1SRVlrMVVRelJYY1V0TU1UZEVkMUJxWWpReVUySkxXbEprWVZwT2FVZFNXRWR0VFRoR2JtdGtUU0lzSW1GMVpDSTZJbVJwWkRwclpYazZlalpOYTJkaFFXRldVM3BGT1ZoeFlXNXlZa1pFTVVvNVVrZFVSbU5YVlRkT09VSlFaMnBUVjIweE4zQnJNVzFDSWl3aVpYaHdJam8wT0RBek9Ea3dOVFV5TENKaGRIUWlPbHRkTENKd2NtWWlPbHRkZlEuUGxQbWllRjdxMEUtTnJkdUhTRkluRHVXQVhwbXR1UzBwdXpwLWpuV1ZkQ1J4Rnphc2l6SlRPMlo4LWxtV0I3R0M5bXYxMV90dHNKVURadVpHYUpZQUEiXX0.NY3FNqTvBl-4uiQcAKzPpDIvBBw5QBrdin_YvYvBAs7ocAHLWfhEWgR8K4aOYw17uKJclzoRbYeNT-i1lgamDg",
+    "assertions": {
+      "header": {
+        "alg": "EdDSA",
+        "typ": "JWT",
+        "ucv": "0.8.1"
+      },
+      "payload": {
+        "iss": "did:key:z6MkgaAaVSzE9XqanrbFD1J9RGTFcWU7N9BPgjSWm17pk1mB",
+        "aud": "did:key:z6MkrjzAyNQV2HmFRiLhNoiTW7RZ76JUTnuHtfsTN1RK6ZPT",
+        "exp": 4803890552,
+        "att": [],
+        "prf": [
+          "eyJhbGciOiJFZERTQSIsInR5cCI6IkpXVCIsInVjdiI6IjAuNyJ9.eyJpc3MiOiJkaWQ6a2V5Ono2TWtwemREYk1UQzRXcUtMMTdEd1BqYjQyU2JLWlJkYVpOaUdSWEdtTThGbmtkTSIsImF1ZCI6ImRpZDprZXk6ejZNa2dhQWFWU3pFOVhxYW5yYkZEMUo5UkdURmNXVTdOOUJQZ2pTV20xN3BrMW1CIiwiZXhwIjo0ODAzODkwNTUyLCJhdHQiOltdLCJwcmYiOltdfQ.PlPmieF7q0E-NrduHSFInDuWAXpmtuS0puzp-jnWVdCRxFzasizJTO2Z8-lmWB7GC9mv11_ttsJUDZuZGaJYAA"
+        ]
+      },
+      "validationErrors": [
+        "prfWitnessVersionMismatch"
+      ]
+    }
+  },
+  {
+    "comment": "Witness does not exist",
```

Comment:
This comment could be made more specific by mentioning proof schemes. One possibility would be "Witness referenced in prf scheme does not exist".